src/editDisplayUtils.ts

```diff
@@ -44,8 +44,34 @@ export default async function displayPendingEditDecorations(
 
   await decorationSleep();
 
-  await runOnTargetsForEachEditor(targets, async (editor, selections) => {
+  await runOnTargetsForEachEditor(targets, async (editor) => {
```

Comment:
I guess typescript doesn't complain about the missing parameter?

Reply:
No unused arguments can be left out just as in regular javascript seems. I'm glad for that otherwise you had make your lambdas much larger for each foreach and filter function.